app/views/result/index.html.erb

```diff
@@ -30,16 +36,19 @@
           <label class="form-label" for="result_count">
             Required number of results
           </label>
-          <input class="form-control" id="result_count" name="count" type="number"></input>
+          <input value="<%= params["count"] %>" class="form-control" id="result_count" name="count" type="number"></input>
```

Comment:
Thanks for this one, had no idea there were a thing, I'll change the welcome file too